### DISCUSSION TITLE
I don't have write permission to the target environment.

### DISCUSSION THREAD

**Discussion by Geozike** [giswqs/geog-312#63] [Q&A] [answered]
I am trying to install mamba in my base environment but keep on receiving this error. That, I don't have write permission to the target environment. Can you please help me out sir.
![Problem 1](https://github.com/user-attachments/assets/7d9497e7-28ab-4700-989c-dd74258b9039)

**Comment by giswqs** [comment, marked as answer]
You can remove `-n base` to just install mamba into the env where you have write permission. 

```python
conda create -n geo python=3.11
conda activate geo
conda install mamba -c conda-forge
```

**Reply by Geozike** [reply]
thank you sir
